requirements/common.txt

```diff
@@ -1,31 +1,31 @@
-beautifulsoup4==4.4.0
-django-admin-sortable2==0.6.4
+beautifulsoup4==4.4.1
+django-admin-sortable2==0.6.7
 django-allauth==0.23.0
 django-angular==0.8.4
 django-appconf==1.0.1
 django-classy-tags==0.7.2
-django-cms==3.3.3
+https://github.com/jrief/django-cms/archive/f9b62031f64aa7d775f9d1befa7b6a70d0d1a719.zip#egg=django-cms
```

Comment:
That's the problem I reported two weeks ag: https://github.com/divio/django-cms/pull/5809
Fortunately now it seems, as if the rejection of this patch is withdrawn, status was changed into "needs design decision".

Therefore this pull request shall be considered as step to the next release, presumably 0.10.
Nevertheless, it should become the master branch on which we can work upon.